contracts/external/IUniswapV2Router.sol

```diff
@@ -0,0 +1,95 @@
+pragma solidity >=0.6.2;
+
+interface IUniswapV2Router {
```

Comment:
Something changed in @uniswap dep and I couldn't compile locally with their import.. This is just a cp/paste of the interface